spring-boot/ServletReg/README.md

```diff
@@ -0,0 +1,13 @@
+# Servlet Reg
```

Comment:
I am not sure what this file is for.
It's a good idea to have a README file here, but let's make it readable.
In other words, don't abbreviate words, and use proper Markdown syntax for the bullet points under 'Specifics'.